api/internal/gateway/admin/v1/handler/live.go

```diff
@@ -31,6 +33,28 @@ func (h *handler) GetLive(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, res)
 }
 
+func (h *handler) UpdateLivePublic(ctx *gin.Context) {
+	req := &request.UpdateLivePublicRequest{}
+	if err := ctx.BindJSON(req); err != nil {
+		badRequest(ctx, err)
+		return
+	}
+
+	in := &store.UpdateLivePublicInput{
+		LiveID:      util.GetParam(ctx, "liveId"),
+		Published:   req.Published,
+		Canceled:    req.Canceled,
```

Comment:
PRコメントに書いてくれてるように、Gateway側ではここは固定値で入れるようにしていいかも👀

```suggestion
		Published:   true,
		Canceled:    false,
```